### PR TITLE
[docs] Update explain and fix bug

### PR DESCRIPTION
**Pull request by KassieZ**
## Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [ ] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [x] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/sql-manual/sql-statements/data-query/EXPLAIN.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+| FRAGMENT  | 执行片段。由于 DORIS 是一个分布式执行引擎。一个完整的执行计划会被切分为多个单机的执行片段。一个 FRAGMENT 表是一个完整的单机执行片段。多个 FRAGMENT 组合在一起，构成一个完整的 PLAN。 |
+| PLAN NODE | 算子。执行计划的最小单位。一个 FRAGMENT 由多个算子构成。每一个算子负责一个实际的执行逻辑，比如聚合，连接等。 |
+
+### 返回结果结构
+
+Doris EXPLAIN 语句的结果是一个完整的 PLAN。PLAN 内部是按照执行顺序从后到前有序排列的 FRAGMENT。FRAGMENT 内部是按照执行顺序从后到前有序排列的算子（PLAN NODE）。
+
+示例如下：
+
+```sql
```
Comment (morrySnow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/sql-manual/sql-statements/data-query/EXPLAIN.md:68]: 这里不要sql把，plain text 就可以

**Review by morrySnow — APPROVED** [review]

File: docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+
+```sql
+{EXPLAIN | DESC} [VERBOSE] <query_block>
+```
+
+## Required Parameters
+
+**<query_block>**
+
+> The query statement for which you want to view the execution plan.
```
Comment (morrySnow) [inline docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md:47]: This is the query statement for which you want the explain plan.

File: docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+| Sink               | The method of fragment data output, see the table below for details |
+
+
+
+**Sink Methods**
+
+
+| Name               | Description                                                  |
+| :----------------- | :----------------------------------------------------------- |
+| STREAM DATA SINK   | Outputs data to the next Fragment. It includes two lines of information.<br />First line: The downstream EXCHANGE NODE to which data is sent.<br />Second line: The method of data distribution. UNPARTITIONED means each downstream instance receives the full data set.<br />This typically occurs in broadcast joins or when single-instance logic is required, such as global limit or order by.<br />RANDOM means each downstream instance receives a random subset of data without repetition.<br />HASH_PARTITIONED uses the listed slots as keys to hash and send data shards to the same downstream instance. This is often used upstream of partition hash joins or the second stage of two-phase aggregations. |
```
Comment (morrySnow) [inline docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md:149]: 这里的换行有些问题
```
Second Line
 - UNPARTITIONED
 - RANDOM
 - HASH_PARTITIONED
```

File: docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md [hunk 4]
```diff
@@ -0,0 +1,658 @@
...
+```
+
+#### TupleDescriptor
+
+
+
+| Name | Description                                                  |
+| :--- | :----------------------------------------------------------- |
+| id   | The id of the tuple descriptor                               |
+| tbl  | The corresponding table of the tuple, or left blank if not applicable |
```
Comment (morrySnow) [inline docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md:176]:
```suggestion
| tbl  | The corresponding table of the tuple, or 1null` if not applicable |
```

File: docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md [hunk 5]
```diff
@@ -0,0 +1,658 @@
...
+
+
+
+#### SlotDescriptor
+
+| Name            | Description                                                  |
+| :-------------- | :----------------------------------------------------------- |
+| id              | The id of the slot descriptor                                |
+| col             | The corresponding column of the slot, or left blank if not applicable |
```
Comment (morrySnow) [inline docs/sql-manual/sql-statements/Data-Manipulation-Statements/Manipulation/EXPLAIN.md:185]: null

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/sql-manual/sql-statements/data-query/EXPLAIN.md (hunk 1 above)
Comment (KassieZ) [reply i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/sql-manual/sql-statements/data-query/EXPLAIN.md:68]: OK

**Pull request merged**